seqr/views/utils/variant_utils.py

```diff
@@ -161,8 +175,10 @@ def _add_pa_detail(locus_list_gene, locus_list_guid, gene_json):
 LOAD_PROJECT_TAG_TYPES_CONTEXT_PARAM = 'loadProjectTagTypes'
 LOAD_FAMILY_CONTEXT_PARAM = 'loadFamilyContext'
 
+
 def get_variants_response(request, saved_variants, response_variants=None, add_all_context=False, include_igv=True,
-                          add_locus_list_detail=False, include_rna_seq=True, include_project_name=False):
+                          add_locus_list_detail=False, include_rna_seq=True, include_project_name=False,
+                          include_phenotype_prioritization=True):
```

Comment:
Is there any place in the code where we want one of rna seq and phenotype prioritization but not both? If not, we should change `include_rna_seq` to be `include_individual_gene_scores` and use one flag to control the behavior for this